README.md

```diff
@@ -1,14 +1,63 @@
 ![raspap-docker-repository](https://user-images.githubusercontent.com/229399/111151581-edb7df00-858f-11eb-8e3a-3ac11c3c04b7.png)
 
-
 # raspap-docker
+
 A community-led docker container for RaspAP
 
-# Usage
+## Setting up using Docker
+
+### Debian 12
+
+1. Start the container with `jrcichra/raspap-docker:bookworm` image
+
+```
+docker run --name raspap -it -d --privileged --network=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cap-add SYS_ADMIN jrcichra/raspap-docker:bookworm
```

Comment:
I believe this image would be out of date. The build pipeline is probably building an image on ghcr.io/raspap/raspap-docker, which would be preferable to my docker.io account.

Reply:
@jrcichra Changed the image path